src/plugins/timelist/TimelistComponent.vue

```diff
@@ -375,7 +376,9 @@ export default {
         this.pastActivitiesCount = this.pastActivitiesCount + 1;
       }
 
-      if (!activity.key) {
+      if (activity.id) {
+        activity.key = activity.id;
+      } else if (!activity.key) {
```

Comment:
what's the purpose?

Reply:
For v-for keys

Reply:
Yeah, that's because the VIPER planning tool sends us an `id` parameter already and I didn't want to ask them to change their code. But I talked to them and they'll change it. I'll simplify this.